CHANGELOG.md

```diff
@@ -916,6 +916,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   * Initial release
 
 [Unreleased]: https://github.com/Behat/Behat/compare/v3.7.0...master
+### Changed
```

Comment:
This shouldn't go here, you can remove it and we'll add as part of the release

Reply:
Okido, removed. I followed https://github.com/Behat/Behat/blob/master/CONTRIBUTING.md :thinking:

Reply:
Yeah it's meant to go at the top